backend/flask_api.py

```diff
@@ -201,5 +201,112 @@ def search():
         return (str(e), 400)
 
 
+@app.route('/api/projects/search/simple/<int:offset>/<int:count>/', methods=['GET'])
+def search_simple(offset, count):
+    """Search projects
+
+    Returns:
+        res (json): JSON containing Projects and metadata
+
+    """
+    text = request.args.get("q", type=str)
+    sorting = request.args.get('sort', type=str)
+    order = request.args.get('order', type=str)
```

Comment:
Please don't mix `@app.route` arguments and `request.args`.
Use `@app.route` for a path (like /search or /projects/<id>) and request.args for actual arguments with sensible defaults, where applicable.

So please move `offset` and `count` here as well